### PR TITLE
feat(kubernetes): add namespace to k8s error reporting table

### PR DESCRIPTION
**Pull request by noisyscanner**
## Hey, I just made a Pull Request!

Adds the namespace field to the Kubernetes error reporting table. Closes https://github.com/backstage/backstage/issues/12565

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] ~Added or updated documentation~ Not neccessary
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/flat-zebras-draw.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```
Comment (benjdlambert) [inline .changeset/flat-zebras-draw.md:2]: This can be `patch` for now as it's `<1.0.0`

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Just one things about the changeset, but looks good otherwise! Thanks :tada:

File: .changeset/flat-zebras-draw.md (hunk 1 above)
Comment (noisyscanner) [reply .changeset/flat-zebras-draw.md:2]:
```suggestion
'@backstage/plugin-kubernetes': patch
```

**Review by benjdlambert — APPROVED** [review]
Nice :Tada:

**Comment by suuus** [comment]
Hey @noisyscanner, congrats on your first merged PR 🎉 Nice work 👍 Please let me know if we can be helpful in any way.

**Pull request merged**